src/black/__init__.py

```diff
@@ -6748,9 +6748,10 @@ def fix_docstring(docstring: str, prefix: str) -> str:
     # https://www.python.org/dev/peps/pep-0257/#handling-docstring-indentation
     if not docstring:
         return ""
-    # Convert tabs to spaces (following the normal Python rules)
+    # Convert leading tabs to spaces (following the normal Python rules)
     # and split into a list of lines:
-    lines = docstring.expandtabs().splitlines()
+    docstring_expanded = re.sub(r"^\t+", "\t".expandtabs(), docstring, flags=re.M)
```

Comment:
If there are multiple tabs at the beginning, shouldn't they be replaced by multiple instances of `"\t".expandtabs()`?

Reply:
Oh, duh. Will fix.